### DISCUSSION TITLE
Unable to create a repository for my old Git account due to the error: "The repository ap1781994 already exists on this account."

### DISCUSSION THREAD

**Discussion by ap1781994** [community/community#121863] [Repositories] [answered]
### unable to manage git profile for existing account

Bug

### Unable to create a repository for my old Git account due to the error: "The repository ap1781994 already exists on this account." #121863

![image](https://github.com/community/community/assets/59976398/09ca1028-f136-47fa-be42-9110418acd5d)

I intended to create the repository for the purpose of hosting a GitHub profile README file, as per the guidelines found online. According to these instructions, to initiate the process, I needed to establish a public repository named precisely after my GitHub username and include a file named README.md within it. This README file, once created, would automatically display on my GitHub profile page. However, I encountered an issue when attempting to create the repository for my old Git account. The process halted with an error message stating that the repository with the name "ap1781994" already exists on this account.

Note :  I don't have existing ap1781994 repo in my git 

Thank You

**Comment by carneiroigor** [comment, marked as answer]
Hi, I took a look at your github profile and there is an existing empty repository with your nickname, can you access it?
![Captura de tela de 2024-05-01 18-49-59](https://github.com/community/community/assets/147322237/94cffd56-7be6-47cd-b327-cb45c1a88767)

**Reply by kathanxx** [reply]
Lately, I have been experiencing the same issue. Can you please help me out with it. Tried checking via the URL but shows 404

error while creating the repo  : The repository kathanxx has been retired and cannot be reused.

**Reply by carneiroigor** [reply]
Hi, have you ever changed your username? If so, you need to contact [Github Support](https://support.github.com/request)!

But if that's not your current problem, let me know.

**Reply by kathanxx** [reply]
no , its always been the same

**Reply by carneiroigor** [reply]
Researching a little more about your problem and trying to understand what could have happened, unfortunately I couldn't find any way I can help you, so the best way for you to resolve this, is by talking to [Github Support](https://support.github.com/request).

**Reply by kathanxx** [reply]
okay thanks, generated a ticket

**Comment by ap1781994** [comment]
Thank you!

"Basically, the default repository is in a hidden format. If we manually enter the path of this repository in a browser, we can access the hidden repository."
